File: abstract-factory/go/oracle-command.go
package main

import "fmt"

type oracleCommand struct{}

func newOracleCommand() *oracleCommand {
	return &oracleCommand{}
}

func (c *oracleCommand) executeCommand(query string) {
	// Gelen sorgunun çalıştırılması için
	// Gerekli operasyonel işlemler...
	fmt.Println("Oracle Command Executed")
}

File: abstract-factory/dart/oracle_command.dart
import 'command.dart';

final class OracleCommand implements Command {
  const OracleCommand();

  @override
  void executeCommand(String query) {
    // Gelen sorgunun çalıştırılması için
    // Gerekli operasyonel işlemler...
    print('OracleCommand: $query');
  }
}
